internal/command/darwin.go

```diff
@@ -54,7 +54,7 @@ func (cmd *Darwin) Parse(args []string) error {
 
 	// Add flags to use only on darwin host
 	if runtime.GOOS == darwinOS {
-		flagSet.BoolVar(&cmd.localBuild, "local", false, "If set uses the fyne CLI tool installed on the host in place of the docker images")
+		flagSet.BoolVar(&cmd.localBuild, "local", true, "If set uses the fyne CLI tool installed on the host in place of the docker images")
```

Comment:
If this `localBuild` is set to default to true will it be switched of later if we see that the target OS is not darwin?
It is possible that this could lead to confusion if it does not switch of appropriately.

Reply:
The local flag should be available only on darwin hosts.
Anyway, this is a good point. Wondering if we'd remove it completely until we have full support on macOS.

Reply:
Yes, I see that it is only available on darwin. But won't it impact if we are doing darwin->linux?

Reply:
Oh I see what you meant, sorry misreaded your previous comment.
It won't impact. The logic above applies only to the `darwin` subcommand responsible to compile for the darwin target.

Reply:
Ah that makes sense, thanks